### DISCUSSION TITLE
Unlock smartlink  SWING +  (Octavia 3)

### DISCUSSION THREAD

**Discussion by GhostSVK** [Mr-MIBonk/M.I.B._More-Incredible-Bash#364] [Q&A]
hello, please, is it possible to activate smartlink on the SWING+ radio?  the radio does not have navigation.  I am attaching photos.  please, if anyone has a procedure that will work, direct me.  thank you very much for every response.
![20231126_194724](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/152095205/aa17f6a8-9c6d-4a45-b5c2-b8d4ff7eec3d)
![20231126_194753](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/152095205/71c1f57b-3516-4863-a77c-487ad8c404e5)
![20231126_194824](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/152095205/da9f5e13-7c1b-4c8c-bf5f-83cf59a737bb)

**Comment by You553ph** [comment]
Pleaaaase we are suffering with this radio !
![Uploading IMG_1529.jpeg…]()
